Sources/Apodini/Properties/Binding.swift

```diff
@@ -32,10 +32,12 @@ private enum Retrieval<Value> {
 /// or an `@Environment`. The latter `Binding`s for the latter two are contained in their
 /// `projectedValue`s.
 @propertyWrapper
-public struct Binding<Value>: DynamicProperty, PotentiallyParameterIdentifyingBinding {
+public struct Binding<Value: Codable>: DynamicProperty, PotentiallyParameterIdentifyingBinding {
```

Comment:
Why constrain the `Value` to `Codable`? If we should decide to use `Codable` for collecting `Property`s on `Handler`s, the only option to have a non-`Codable` "regular" property (for parametrization or whatever) on a `Handler` would be through a `Binding`. Thus, I would prefer not to constrain `Binding` here as I think it isn't really necessary.

Reply:
I changed this as the `PathComponentParser`'s `mutating func visit<T>(_ parameter: Parameter<T>)` requires that the type conforms to Codable. I will check how this could be circumvented so we don't need the constraint to `Codable` here.

Reply:
I have addressed the feedback by making the conformance to `PathComponent` and `_PathCompoent` if value conforms to `Codable`:
```swift
extension Binding: PathComponent & _PathComponent where Value: Codable {
    func append<Parser: PathComponentParser>(to parser: inout Parser) {
        guard let parameter = store.wrappedValue["parameter"] as? Parameter<Value> else {
            assertionFailure("Only bindings created from a `Parameter` or `PathParameter` can be used as a path component")
            return 
        }
        
        parser.visit(parameter)
    }
}
```